sdk/iot/core/tests/cmocka/az_iot_telemetry_tests.c

```diff
@@ -0,0 +1,283 @@
+#include <az_iot_hub_client.h>
+#include <az_span.h>
+
+#include <stdio.h>
+
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include <cmocka.h>
+
+#define TEST_CORRECT_TOPIC_PREFIX "devices/"
+#define TEST_CORRECT_TOPIC_SUFFIX "/messages/events/"
+#define TEST_CORRECT_TOPIC_MODULES_MID "/modules/"
+
+#define TEST_FQDN "myiothub.azure-devices.net"
+#define TEST_DEVICE_ID "my_device"
+#define TEST_MODULE_ID "my_module_id"
+#define TEST_USER_AGENT "api-version=2017-11-08-preview&DeviceClientType=iothubclient"
```

Comment:
update the api version?

Reply:
Thats what it is in the current sdk. Directly copied over.